src/modules/mavlink/mavlink_messages.cpp

```diff
@@ -4531,44 +4531,48 @@ class MavlinkStreamDistanceSensor : public MavlinkStream
 
 	bool send(const hrt_abstime t) override
 	{
-		distance_sensor_s dist_sensor;
+		auto updated = false;
```

Comment:
Nitpicky, but I think the general consensus is to not use `auto` for primitives.